src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const relationship = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${relationship}`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
1. Let's add empty lines between sibling elements
2. You always render `<p className="Person__partner">` so put condition inside it
```
<h2 className="Person__name">{`My name is ${name}`}</h2>

{age && <p className="Person__age">{`I am ${age}`}</p>}

<p className="Person__partner">
  {isMarried
    ? `${partnerName} is my ${relationship}`
    : 'I am not married'}
</p>
```